ui/src/components/shared/TopBar.vue

```diff
@@ -586,7 +591,9 @@ function showReleaseNotes() {
 }
 
 function changeTheme(newTheme: string) {
-  PrimeVue.changeTheme(currentTheme.value, newTheme, "theme-link", () => userStore.updateCurrentTheme(newTheme));
+  PrimeVue.changeTheme(currentTheme.value, newTheme, "theme-link", () => {});
+  userStore.updateCurrentTheme(newTheme);
+  currentTheme.value = newTheme;
 }
```

Comment:
Just wondering why this has come out of the lambda function, rather than...
```
  PrimeVue.changeTheme(currentTheme.value, newTheme, "theme-link", () => {
      userStore.updateCurrentTheme(newTheme);
      currentTheme.value = newTheme;
  });
```